tools/migration/migrations/repo-1-2/migration_test.go

```diff
@@ -67,6 +67,12 @@ func TestMigrateSomeRepo(t *testing.T) {
 	require.NoError(t, os.Symlink(repoDir, repoSymLink))
 
 	t.Run("Happy path: valid migration passes validation", func(t *testing.T) {
+		// TODO the upgrade from go-cidv0.0.1 to go-cidv0.0.3 has broken this test
+		// as the encoding has changed from base58 to base32. This causes key
+		// lookups in the chain store to fail since the CID's are no longer
+		// the expected values.
+		// This skip can be removed when go-filecoin#3133 is addressed.
+		t.SkipNow()
```

Comment:
To the reviewer: does this sound reasonable to you?

Reply:
Sure, we'll delete this migration and test eventually. It's functioning primarily as a demonstration.